lib/deaf_grandma.rb

```diff
@@ -3,6 +3,7 @@
 #CAREFUL! This script will not exit. Do you know why? You may have to close it with `Ctrl-C` (Mac) if you do not insert an `exit` into your speak method.
 
 class DeafGrandma
+  attr_accessor :bye_counter
```

Comment:
This defines getter and setter methods for `bye_counter`. You don't need to access the instance method (`@bye_counter`) directly. Basically, you should drop the `@`s.